src/i18n/rpi-imager_zh.ts

```diff
@@ -123,16 +123,16 @@
         <translation>使用自定义镜像</translation>
     </message>
     <message>
-        <source>Cached file is corrupt. SHA256 hash does not match expected value.&lt;br&gt;The cache file will be removed and the download will restart.</source>
-        <translation type="unfinished"></translation>
+        <source>缓存文件已损坏。SHA256 哈希值与预期值不匹配。&lt;br&gt;该缓存文件将被删除，将重新开始下载。</source>
+        <translation></translation>
```

Comment:
Ooops, something went  wrong here (and also a few other places). The original English text needs to remain in the `<source> </source>` tags, and the Chinese translation needs to go in the `<translation> </translation>` tags.

Reply:
<img width="963" height="106" alt="image" src="https://github.com/user-attachments/assets/b8cfbff7-04f6-4f31-8ca7-bc07ff5f7457" />
This seems to have been a previous error, which should now be corrected.